spk/ezstream/Makefile

```diff
@@ -0,0 +1,22 @@
+SPK_NAME = ezstream
+SPK_VERS = 0.6.0
+SPK_REV = 1
+SPK_ICON = src/ezstream.png
+
+DEPENDS  = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = EZStream is a command line source client for Icecast media streaming server 
+
+DISPLAY_NAME = ezstream
+
+HOMEPAGE   = http://www.icecast.org/
+LICENSE    = GPLv2
+
+STARTABLE = no
+
+INSTALLER_SCRIPT = src/installer.sh
```

Comment:
Please remove custom installer script, to take benefits of generic framework installer - as an example of link creation, please have a look at https://github.com/SynoCommunity/spksrc/tree/master/spk/tmux